cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpLogger.java

```diff
@@ -0,0 +1,44 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import okhttp3.logging.HttpLoggingInterceptor;
+
+final class CwmsHttpLogger implements HttpLoggingInterceptor.Logger {
+
+    private static final String ACCESS_TOKEN = "access_token";
+    private static final String REFRESH_TOKEN = "refresh_token";
+    private static final Logger LOGGER = Logger.getLogger(CwmsHttpLogger.class.getName());
+    private static final HttpLoggingInterceptor.Logger DELEGATE = HttpLoggingInterceptor.Logger.DEFAULT;
+
+    @Override
+    public void log(String s) {
+        String lowerCaseS = s.toLowerCase();
+        if (!lowerCaseS.contains(ACCESS_TOKEN.toLowerCase()) && !lowerCaseS.contains(REFRESH_TOKEN.toLowerCase())) {
+            DELEGATE.log(s);
+        } else {
+            try {
+                s = redactOAuth2Tokens(s);
+                DELEGATE.log(s);
+            } catch (JsonProcessingException e) {
+                LOGGER.log(Level.INFO, () -> "Could not parse logged JSON. Redacting log containing token(s) for security reasons.");
```

Comment:
Change this to FINE or FINER, this should only be turned on if you can't find why your values aren't getting logged.

Reply:
This has been removed completely